server/models/Ticket.js

```diff
@@ -1,30 +1,23 @@
-const mongoose = require('mongoose');
+const { Schema } = require("mongoose");
 
-const { Schema } = mongoose;
 
-const ticketSchema = new  Schema({
-    purchaseDate:{
-        type : Date,
-        default : Date.now
-    },
-    price: {
-        type: Number,
-        required: true,
-        min: 0.99
-      },
+const ticketSchema = new Schema({
+  purchaseDate: {
+    type: Date,
+    default: Date.now,
+  },
+  price: {
+    type: Number,
+    required: true,
+    min: 0.99,
+  },
 
-      seatNumber : {
-        type : Number,
-        required : true
+  seatNumber: {
+    type: Number,
```

Comment:
- seatNumber type:String would be better because seat can be somethinglike 5A, 33B, etc.